### DISCUSSION TITLE
wrong values from analog reading

### DISCUSSION THREAD

**Discussion by Bjrow** [jomjol/AI-on-the-edge-device#2492] [Q&A]
Hello
My AI reads an analog number incorrectly.
It's the second analog, the one with the metal element. this unfortunately makes the counter values completely wrong. Is there a better config for this pointer type.

Version: 1.5.24

![Wasserzaehler](https://github.com/jomjol/AI-on-the-edge-device/assets/133246179/6d077542-8b4e-47d8-a3a1-82dac1a88347)

**Comment by ghost** [comment]
Which model do you use?

**Reply by ghost** [reply]
The ROIs are looking good

**Reply by Bjrow** [reply]
Sorry for the long delay....

The model i have used was ana_cont_1105_s2.
Yesterday i have change it to ana_class_100_0157 and i have enlarged the circle for the ROI. So the black circle of the ROI is better in the Area.

At the Moment it's looks like the values are better.

**Comment by Bjrow** [comment]
short update,

with the new model and the enlarged circle the values are good.
